### ISSUE TITLE
Your "pos_weights" method doesn't seem to work 

### ISSUE THREAD

**Issue by maovshao** [flatironinstitute/DeepFRI#11]
In train_DeepCNN.py
`    # computing weights for imbalanced go classes
    class_sizes = counts[args.ontology]
    mean_class_size = np.mean(class_sizes)
    pos_weights = mean_class_size / class_sizes
    pos_weights = np.maximum(1.0, np.minimum(10.0, pos_weights))
    pos_weights = np.concatenate([pos_weights.reshape((len(pos_weights), 1)), pos_weights.reshape((len(pos_weights), 1))], axis=-1)

     model.train(train_tfrecord_fn, valid_tfrecord_fn, epochs=args.epochs, batch_size=args.batch_size, pad_len=args.pad_len, ont=args.ontology, class_weight=pos_weights)
`

Also, in DeepCNN.py
`
        # fit model
        history = self.model.fit(batch_train,
                                 epochs=epochs,
                                 validation_data=batch_valid,
                                 class_weight=class_weight,
                                 steps_per_epoch=n_train_records//batch_size,
                                 validation_steps=n_valid_records//batch_size,
                                 callbacks=[es, mc])
`

But I found that the "pos_weights" and "class_weight" you write here doesn't work and has errors.
"class_weight" seems to be important when it comes to more goterms, do you have some ideas?
Thanks!!! XD

**Comment by comeon01** [comment]
I also encountered this problem，do you have some ideas?
